cloud/src/meta-service/meta_service_tablet_stats.cpp

```diff
@@ -36,7 +36,8 @@
 
 namespace doris::cloud {
 
-void internal_get_tablet_stats(MetaServiceCode& code, std::string& msg, Transaction* txn,
+void internal_get_tablet_stats(MetaServiceCode& code, std::string& msg,
+                               std::unique_ptr<Transaction>& txn, std::shared_ptr<TxnKv>& txn_kv,
```

Comment:
This is a behavior change, semantic changed, txn atomic is not garanteed if we create a new txn.
Fix where this `internal_get_tablet_stats()` is called if we met  TXN_TOO_OLD

Reply:
@gavinchou already change the code, take a look please